airflow/utils/context.pyi

```diff
@@ -102,24 +102,23 @@ class Context(TypedDict, total=False):
     ds: str
     ds_nodash: str
     exception: BaseException | str | None
-    execution_date: DateTime
+    logical_date: DateTime
```

Comment:
Same, this should be sorted.

Also, removed deprecated fields should be removed here too.

Reply:
changed it.